apache-activemq-mixin/dashboards/apache-activemq-cluster-overview.libsonnet

```diff
@@ -0,0 +1,592 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local prometheus = grafana.prometheus;
+
+local dashboardUid = 'apache-activemq-cluster-overview';
+
+local promDatasourceName = 'prometheus_datasource';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local clusterCountPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'count by (activemq_cluster) (activemq_memory_usage_ratio{job=~"$job"})',
+      datasource=promDatasource,
+      legendFormat='{{activemq_cluster}}',
+    ),
+  ],
+  type: 'stat',
+  title: 'Cluster count',
+  description: 'Number of clusters that are reporting metrics from ActiveMQ.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [],
+  },
+  options: {
+    colorMode: 'none',
```

Comment:
<img width="469" alt="image" src="https://github.com/grafana/jsonnet-libs/assets/38274348/76b9d0bd-466f-4cfd-be72-1235372c7e6f">
I followed this style guideline bullet point. Since the numbers are just numbers and they arent signifying any level of urgency I left them white. I believe this is what they meant?

Reply:
i gotcha, yea im not too sure cus in there they're referencing an actual color mode called "Text" but there isn't an option for that ... i think it's fine for now and we'll see if they say anything when they review